app/controllers/quotes_controller.rb

```diff
@@ -16,15 +16,20 @@ def index
 
   def create
     authenticate_user!
-    @anime = Anime.find(params["quote"]["anime_id"])
-    @quote = Quote.new
-    @quote.content = params["quote"]["content"]
-    @quote.character_name = params["quote"]["character_name"]
-    @quote.anime = @anime
-    @quote.user = current_user
-    @quote.save
-
-    redirect_to anime_quotes_path(@anime)
+
+    quote_params = params.require(:quote).permit(
+      :anime_id, :character_name, :content, :favorite_count, :username
```

Comment:
Well, it wouldn't make a difference since the data is not written to the model but I should probably remove that from there after copying the params from request :frowning: 
Will fix and squas.